challenge_2/python/ning/challenge_2.py

```diff
@@ -0,0 +1,20 @@
+def find_unique(sequence):
+    item_counter = dict()
```

Comment:
Hi, thanks for your input. I previously used `list() or dict()` because I thought they were more readable. But considering that `[] or {}` are both more conventional and efficient, I will follow your advice.

I have also changed the code to use defaultdict in 0600f3a. I did not previously know about this module, but now I do.